units/pod_termination.go

```diff
@@ -202,8 +203,14 @@ func (j *podTerminationJob) populateIfUnset(ctx context.Context) error {
 // tasks in the dispatcher as needing re-allocation.
 func (j *podTerminationJob) fixStrandedTasks(ctx context.Context) error {
 	if j.pod.RunningTask != "" {
+		// A stranded task will need to be re-allocated to ensure that it
+		// dispatches to a new pod after this one is terminated.
+		if err := task.MarkManyContainerDeallocated([]string{j.pod.RunningTask}); err != nil {
+			return errors.Wrapf(err, "marking stranded container task '%s' running on pod as deallocated", j.pod.RunningTask)
```

Comment:
can we add `pod '%s'` to this message?

Reply:
Done. Also added it to the other `errors.Wrap`s in this functions.